src/radians.rs

```diff
@@ -34,6 +34,11 @@ impl Radians {
     pub fn value(self) -> f64 {
         self.value
     }
+
+    /// Try getting the radians value of the degrees
+    pub fn try_from_degrees(degrees: f64) -> Result<Radians, RadiansError> {
```

Comment:
Should we call it `try_new_from_degrees`?

Reply:
`from_degrees`, follow the [convention of the standard library](https://doc.rust-lang.org/nightly/std/string/struct.String.html#method.from_utf8)

Reply:
I think `try_from` and `new` are mutually exclusive in a function name…

Reply:
`from_degrees` seems sensible to me.

Reply:
Aw fish! You were faster @jnferner.